plugins/splunk-on-call/schema.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  /**
+   * Splunk On Call Plugin specific configs
+   */
+  splunkOnCall: {
```

Comment:
Would be this tweak that we're looking for:

```suggestion
  splunkOnCall?: {
```

Or if you enable edit on PR we can apply the fix too in case your org allows it